src/TForkedProcess.cpp

```diff
@@ -91,14 +91,14 @@ int TForkedProcess::sendMessage(lua_State* L)
 {
     QPointer<TForkedProcess>* forkedProcess = *((QPointer<TForkedProcess>**)lua_topointer(L, lua_upvalueindex(1)));
 
-    if (forkedProcess == 0 || forkedProcess->isNull() || !(*forkedProcess)->running) {
+    if (forkedProcess == nullptr || forkedProcess->isNull() || !(*forkedProcess)->running) {
```

Comment:
I think `if ((!forkedProcess) || forkedProcess->isNull() || !(*forkedProcess)->running) {` would be more apt here.